updates for IDX content (#10107)
withastro/docs: src/content/docs/en/tutorial/1-setup/index.mdx

This unit shows you how to set up your development environment and deploy to Netlify. Skip ahead to [Unit 2](/en/tutorial/2-pages/) if you are already comfortable with your environment and workflow.

:::tip[Take the tutorial in an online code editor]

> Want to complete this tutorial in an online code editor instead? Follow the instructions below for either IDX or StackBlitz.

Author: ```suggestion
Want to complete this tutorial in an online code editor instead? Follow the instructions below for either Google IDX or StackBlitz.
```

<details>

> <summary>Using IDX: Follow these instructions, then go directly to Unit 2!</summary>

Author: ```suggestion
<summary>Using Google IDX: Follow these instructions, then go directly to Unit 2!</summary>
```

**Set up IDX**

<Steps>

> 1. Visit [astro.new](https://astro.new) and click the button to open the "Empty Project" template in IDX. 

Author: ```suggestion
1. Follow the external link to [open the "Empty Project" template in a new workspace on IDX](https://astro.new/minimal?on=idx).
```

3. Follow the prompts to sign in to your GitHub account.
4. Once you are signed in, return to the IDX tab and you will be given the choice to name your new repository, and whether you want to create a private or public repository. You can choose any name and either kind of repository for this tutorial.
5. IDX will make an initial commit and publish to your new GitHub repo.
6. Going forward, whenever you have changes to be committed back to GitHub, the Source Control navigation icon will show a number. This is the number of files that have changed since your last commit. Navigating to this tab and performing two steps (commit and publish) will allow you to enter a commit message, and update your repository.
</Steps>

**Deploy your Site**

If you'd like to deploy to Netlify, and have a live published version of your site while you work, go ahead in Unit 1 to [Deploy your site to the web](/en/tutorial/1-setup/5/).

Otherwise, skip to [Unit 2](/en/tutorial/2-pages/) to start building with Astro!

</details>

<details>
<summary>Using StackBlitz: Follow these instructions, then go directly to Unit 2!</summary>

**Set up StackBlitz**

<Steps>

> 1. Visit [astro.new](https://astro.new) and use the drop down select to open the "Empty Project" template in StackBlitz. 

Author: ```suggestion
1. Follow the external link to [open the "Empty Project" template on StackBlitz](https://astro.new/minimal?on=stackblitz).
```
